ooni/nettest.py

```diff
@@ -541,6 +561,13 @@ def doneReport(self, report_results):
         The state for the NetTest is informed of the fact that another task has
         reached the done state.
         """
+        self._completedInputs += 1
+        log.msg("Status")
+        log.msg("------")
+        log.msg("%d completed %d remaining" % (self._completedInputs,
+                                               self._totalInputs))
+        log.msg("%0.1f%% (ETA: %ds)" % (self.completionPercentage * 100,
+                                        self.completionEta))
```

Comment:
There is something wrong in the way in which ETA is computed. From my console logs:

```
Status
------
4 completed 0 remaining
0.0% (ETA: -4s)
```

Note: the test here was http-invalid-request-line.